grim.go

```diff
@@ -117,10 +117,21 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 		if err != nil {
 			return grimErrorf("error extracting hook from message: %v", err)
 		}
+		logger.Print(hook.Describe() + "\n")
```

Comment:
Why use an interface rather than just a method? The only thing the we would want from it is the string